interwovenkit/features/autosign/configuration.mdx

```diff
@@ -69,54 +70,29 @@ Before configuring your application, set up allowed domains in the Privy Dashboa
 
 </Step>
 
-<Step title="Configure Privy Provider">
+<Step title="Configure Privy and Integrate with InterwovenKit">
 
-Wrap your application with `PrivyProvider` and configure it for embedded wallets:
-
-```tsx App.tsx
-import { PrivyProvider } from "@privy-io/react-auth"
-import { InterwovenKitProvider } from "@initia/interwovenkit-react"
-
-export default function App({ children }) {
-  return (
-    <PrivyProvider
-      appId={process.env.NEXT_PUBLIC_PRIVY_APP_ID}
-      config={{
-        embeddedWallets: {
-          createOnLogin: "all-users",
-        },
-      }}
-    >
-      {/* Your app */}
-    </PrivyProvider>
-  )
-}
-```
+Wrap your app with `PrivyProvider`, then pass Privy hooks into `InterwovenKitProvider` using `privyContext`. This enables embedded wallets and autosign support.
```

Comment:
Do you find the code example not necessary here?

If we decide to omit the example, we should still specify that they need to set `embeddedWallets.ethereum.createOnLogin` to `all-users` in the `PrivyProvider` config since the default seems to be `off` (https://docs.privy.io/basics/react/advanced/automatic-wallet-creation#automatic-wallet-creation)